src/app/(Home)/(Domain)/domains/[domain]/[cluster]/[domainTab]/error.tsx

```diff
@@ -5,9 +5,9 @@ import { type RequestError } from '@/utils/request/request-error';
 export default function DomainTabsError({
   error,
   reset,
-}: Readonly<{
+}: {
```

Comment:
Why is it no longer read-only? Only for mocks?

Reply:
I'll probably need to discuss this with Assem once he was back, but the Next.js error page examples in the docs did not mark it as readonly.

In hindsight, I think I'll revert this change.

Reply:
I'm not sure it makes a difference, since we don't try to mutate these properties in the component anyway. But I guess it must have been added for a reason.

Reply:
Added it back